test/e2e/apps/statefulset.go

```diff
@@ -566,7 +574,8 @@ var _ = SIGDescribe("StatefulSet", func() {
 
 		})
 
-		It("should implement legacy replacement when the update strategy is OnDelete", func() {
+		// This is ok for Conformance because it doesn't use any PVCs.
+		framework.ConformanceIt("should implement legacy replacement when the update strategy is OnDelete", func() {
```

Comment:
Don't make this conformance. It's not clear that OnDelete will continue to be a supported update strategy after extensions/v1beta1 is removed.